proto/repositories.bzl

```diff
@@ -16,11 +16,14 @@
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-load("//proto/private:dependencies.bzl", "dependencies")
+load("//proto/private:dependencies.bzl", "dependencies", "maven_dependencies", "protobuf_workspace")
 
 def rules_proto_dependencies():
     for name in dependencies:
         maybe(http_archive, name, **dependencies[name])
+    for name in maven_dependencies:
+        maybe(native.maven_jar, name, **maven_dependencies[name])
```

Comment:
`native.maven_jar` is deprecated and, AFAICT, is removed in Bazel 2.0.
The preferred way of declaring Java dependencies seems to be https://github.com/bazelbuild/rules_jvm_external these days, but I don't think we can use that here. Hopefully, the federation will solve this issue for us.

//cc @fweikert who seems to be working on integrating `rules_proto` into the federation.

Reply:
Switched to java_import_external